content/about/limits.md

```diff
@@ -1,44 +1,78 @@
 
-Accounts using the Workers free tier are limited to a maximum of 100,000 requests per day, with a burst rate limit of 1000 requests per 10 minutes. This limit applies at the account level, meaning that requests on your workers.dev subdomain count toward the same limit as your zoned domains. Visitors who run into the rate limit will be served a Cloudflare 1015 error page, however if you are calling your script programmatically, you can detect the rate limit page and handle it yourself by looking for HTTP status code 429. Upgrading to a paid plan will automatically lift this limit.
+The burst rate and daily request limits apply at the account level, meaning that requests on your workers.dev subdomain count toward the same limit as your zones. Upgrade to a paid plan to automatically lift these limits.
 
-## CPU/Execution Time Limit
+### Burst Rate Limit
+
+Accounts using the Workers *free* tier are subject to a burst rate limit of **1000 requests per 10 minutes**. Users visiting a rate limited site will receive a Cloudflare 1015 error page. However if you are calling your script programmatically, you can detect the rate limit page and handle it yourself by looking for HTTP status code 429.
+
+### Daily Request Limit
+
+Accounts using the Workers *free* tier are subject to a **daily request limit of 100,000 requests**. Free tier daily requests counts reset at midnight UTC. A Worker that fails as a result of daily request limit errors can be configured by setting its corresponding [route](/about/routes/) in two modes: _Fail open_ and _Fail closed_.
 
-| Plan             | CPU  |
-| ---------------- | ---- |
-| Free             | 10ms |
-| Unlimited (Paid) | 50ms |
+#### Fail Open
+
+Routes in fail open mode will bypass the failing Worker and prevent it from operating on incoming traffic. Incoming requests will behave as if there was no Worker.
+
+#### Fail Closed
+
+Routes in fail closed mode will display a Cloudflare 1027 error page to visitors, signifying the Worker has been temporarily disabled. We recommend this option if your Worker is performing security related tasks.
+
+<!-- Add workers request limit error code -->
+
+## CPU/Execution Time Limit
 
-Most Workers requests consume less than a millisecond. It’s rare to find a normally operating Workers script that exceeds the CPU time limit. The 10ms the Free plan allows is enough runtime for most use cases, including application hosting.
+Most Workers requests consume less than a millisecond. It’s rare to find a normally operating Workers script that exceeds the CPU time limit. A Worker may consume up to **10ms on the free tier** and **50ms on the Unlimited tier**. The 10ms allowance on the Free plan is enough execution time for most use cases including application hosting.
```

Comment:
> Most Workers requests consume less than a millisecond

Most Workers executions consume less than a millisecond of CPU

Also - is this true? If a worker is doing anything useful it'll probably be a bit bigger than this..

Reply:
Also we say "Free plan" capitalized here again